README.md

```diff
@@ -1,4 +1,27 @@
-# Python boilerplate for GitHub tasks
+# Create a file every second
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+
+Write an app that will create 1 file every 1 second in current directory (where the script is located).
+
+- File name must be in the following format: `app-{hours}_{minutes}_{seconds}.log`.
+- File content must be a timestamp of this operation (example: 2007-06-29 13:49:40.059821).
+- The app must print to console timestamp and newly created file name when it completes file creation successfully.
+- The app must run forever until you terminate the process.
+
+`datetime.now()` from module `datetime` and `sleep()` from module `time` can be
```

Comment:
Could you write it as a list:
* Use `datetime.now()` from module `datetime` to get current time
* ......